src/parse.rs

```diff
@@ -277,9 +277,18 @@ impl<'a> Bytes<'a> {
     }
 
     pub fn identifier(&mut self) -> Result<&'a [u8]> {
-        if IDENT_FIRST.contains(&self.peek_or_eof()?) {
-            let bytes = self.next_bytes_contained_in(IDENT_CHAR);
+        let next = self.peek_or_eof()?;
+        if IDENT_FIRST.contains(&next) {
+            // If the next two bytes signify the start of a raw string literal,
+            // return an error.
+            if next == b'r' {
```

Comment:
I'm not quite sure why we need this extra check here, sorry.

Reply:
Forgot to delete this comment, it makes sense with the current code, but I'm not sure it's the right way.

Reply:
@torkleyy The only reason I added this branch is to only perform the second byte if the next byte is an `r`. However, I agree that it may not be the best way. This was the best solution I could come up with, however, given the current code structure. If you have any better ideas, please let me know and I'll implement them!